src/Services/ShipmentRegisterService.php

```diff
@@ -268,6 +269,17 @@ public function register(array $orderIds): array
 
         try {
           $shippingInformation = $this->getOrderShippingInformation($orderId);
+          $this->loggerContract->debug(
+            TranslationHelper::getLoggerKey(self::LOG_KEY_DEBUG_REGISTER_INFO),
+            [
+              'additionalInfo' => [
+                'orderID' => $orderId,
+                'message' => 'ShipmentRegister service line 294',
+                'shippingInformation' => $shippingInformation
+              ],
+              'method' => __METHOD__
+            ]
+          );
```

Comment:
Looks like overly-specific leftover debug code (with line numbers).

Reply:
This was to help with pin-pointing the specific log I will generalize